collectors/node/cpu.go

```diff
@@ -167,3 +167,11 @@ func round(f float64) float64 {
 	shift := math.Pow(10, float64(2))
 	return math.Floor(f*shift+.5) / shift
 }
+
+// Helper function: replace all negative numbers with zero
+func gtZero(n float64) float64 {
+	if math.Signbit(n) {
```

Comment:
I thought it was better style than a simple less-than check. Would you prefer `n < 0`?